COBOL doesn't build on windows msvc

Hoist by my own petard when building Tracey on Windows, haha. COBOL has a big sad.

```
parser.c
  scanner.c
  C:\Users\admin\.cargo\registry\src\index.crates.io-1949cf8c6b5b557f\arborium-cobol-2.14.0\grammar\scanner.c(40): error C2057: expected constant expression
  C:\Users\admin\.cargo\registry\src\index.crates.io-1949cf8c6b5b557f\arborium-cobol-2.14.0\grammar\scanner.c(40): error C2466: cannot allocate an array of constant size 0
  C:\Users\admin\.cargo\registry\src\index.crates.io-1949cf8c6b5b557f\arborium-cobol-2.14.0\grammar\scanner.c(40): error C2133: 'keyword_pointer': unknown size
  C:\Users\admin\.cargo\registry\src\index.crates.io-1949cf8c6b5b557f\arborium-cobol-2.14.0\grammar\scanner.c(41): error C2057: expected constant expression
  C:\Users\admin\.cargo\registry\src\index.crates.io-1949cf8c6b5b557f\arborium-cobol-2.14.0\grammar\scanner.c(41): error C2466: cannot allocate an array of constant size 0
  C:\Users\admin\.cargo\registry\src\index.crates.io-1949cf8c6b5b557f\arborium-cobol-2.14.0\grammar\scanner.c(41): error C2133: 'continue_check': unknown size
  exit code: 0
  exit code: 2
  cargo:warning=ToolExecError: command did not execute successfully (status code exit code: 2): "C:\\Program Files\\Microsoft Visual Studio\\2022\\Community\\VC\\Tools\\MSVC\\14.42.34433\\bin\\HostX64\\x64\\cl.exe" "-nologo" "-MD" "-O1" "-Brepro" "-I" "C:\\Users\\admin\\.cargo\\registry\\src\\index.crates.io-1949cf8c6b5b557f\\arborium-cobol-2.14.0\\grammar/src" "-I" "C:\\Users\\admin\\.cargo\\registry\\src\\index.crates.io-1949cf8c6b5b557f\\arborium-cobol-2.14.0\\grammar" "-I" "C:\\Users\\admin\\.cargo\\registry\\src\\index.crates.io-1949cf8c6b5b557f\\arborium-cobol-2.14.0\\grammar/src\\tree_sitter" "-W0" "-FoC:\\Users\\admin\\AppData\\Local\\Temp\\cargo-installgbuArb\\release\\build\\arborium-cobol-5f249b7e53f7ab8e\\out\\ee7f52c1a7335d1c-scanner.o" "-c" "C:\\Users\\admin\\.cargo\\registry\\src\\index.crates.io-1949cf8c6b5b557f\\arborium-cobol-2.14.0\\grammar\\scanner.c"

  --- stderr


  error occurred in cc-rs: command did not execute successfully (status code exit code: 2): "C:\\Program Files\\Microsoft Visual Studio\\2022\\Community\\VC\\Tools\\MSVC\\14.42.34433\\bin\\HostX64\\x64\\cl.exe" "-nologo" "-MD" "-O1" "-Brepro" "-I" "C:\\Users\\admin\\.cargo\\registry\\src\\index.crates.io-1949cf8c6b5b557f\\arborium-cobol-2.14.0\\grammar/src" "-I" "C:\\Users\\admin\\.cargo\\registry\\src\\index.crates.io-1949cf8c6b5b557f\\arborium-cobol-2.14.0\\grammar" "-I" "C:\\Users\\admin\\.cargo\\registry\\src\\index.crates.io-1949cf8c6b5b557f\\arborium-cobol-2.14.0\\grammar/src\\tree_sitter" "-W0" "-FoC:\\Users\\admin\\AppData\\Local\\Temp\\cargo-installgbuArb\\release\\build\\arborium-cobol-5f249b7e53f7ab8e\\out\\ee7f52c1a7335d1c-scanner.o" "-c" "C:\\Users\\admin\\.cargo\\registry\\src\\index.crates.io-1949cf8c6b5b557f\\arborium-cobol-2.14.0\\grammar\\scanner.c"
```

--- a/langs/group-rowan/cobol/def/grammar/scanner.c
+++ b/langs/group-rowan/cobol/def/grammar/scanner.c
@@ -19,7 +19,8 @@ static bool is_white_space(int c) {
     return iswspace(c) || c == ';' || c == ',';
 }
 
-const int number_of_comment_entry_keywords = 9;
+#define MAX_KEYWORDS 9
+const int number_of_comment_entry_keywords = MAX_KEYWORDS;
 char* any_content_keyword[] = {
     "author",
     "installation",
@@ -37,8 +38,8 @@ static bool start_with_word( TSLexer *lexer, char *words[], int number_of_words)
         lexer->advance(lexer, true);
     }
 
-    char *keyword_pointer[number_of_words];
-    bool continue_check[number_of_words];
+    char *keyword_pointer[MAX_KEYWORDS];
+    bool continue_check[MAX_KEYWORDS];
     for(int i=0; i<number_of_words; ++i) {
         keyword_pointer[i] = words[i];
         continue_check[i] = true;

